src/store/listDetails/reducers.js

```diff
@@ -16,18 +16,17 @@ export default (state = initialState, { type, payload }) => {
     case 'GET_LIST_DETAILS_RESPONSE':
       return {
         ...state,
-        ...payload,
+        ids: payload.ids,
+        list_name: payload.list_name,
```

Comment:
You don't need `list_name` if you have normalized data, just grab it from database state